docs/source/setup/environment.rst

```diff
@@ -0,0 +1,98 @@
+Managing environments
+~~~~~~~~~~~~~~~~~~~~~
+
+It is critical that each of your dask workers uses the same set of
+python packages and modules when executing your code, so that Dask
+can function. Upon connecting with a distributed ``Client``, Dask
+will automatically check the versions of some critical packages
+(including Dask itself) and warn you of any mismatch.
+
+Most functions you will run on Dask will require imports. This
+is even true for passing any object which is not a python builtin -
+the pickle serialisation method will save references to imported modules
+rather than trying to send all of your source code.
+
+You therefore must ensure that workers have access to all of the modules
+you will need, and ideally with exactly the same versions.
+
+Single-machine schedulers
+`````````````````````````
+
+If you are using the threaded scheduler, then you do not need to do
+anything, since the workers are in the same process, and objects are
+simply shared rather than serialised and deserialised.
+
+Similarly, if you use the multiprocessing scheduler, new processes
+will be copied from, or launched in the same way as the original process,
+so you only need make sure that you have not changed environment variables
+related to starting up python and importing
+code (such as PATH, PYTHONPATH, ``sys.path``).
+
+If you are using the distributed scheduler on a single machine, this is roughly
+equivalent to using the multiprocessing scgheduler, above, if you are launching
+with ``Client(...)`` or ``LocalCluster(...)``.
+
+However, if you are launching your workers from the command line, then you must
+ensure that you are running in the same environment (virtualenv, pipenv or conda).
+
+The rest of this page concerns only distributed clusters.
+
+Maintain consistent environments
+````````````````````````````````
+
+If you manage your environments yourself, then setting up module consistency
+can be as simple as creating environments from the same pip or conda specification
+on each machine. You should consult the documentation for ``pip``, ``pipenv``
+and ``conda``, whichever you normally use. You will normally want to be as specific
+about package versions as possible, and distribute the same environment file to
+workers before installation.
+
```

Comment:
Maybe should mention the `preload` argument here?

Reply:
Yeah, wasn't sure - should we have an "expert" section?

Reply:
That sounds good.

Reply:
I put it in the following block instead, since the list here is for less ephemeral state